.gitattributes

```diff
@@ -1,2 +1,3 @@
 go.sum linguist-generated
 examples/kubernetes/connectivity-check/connectivity-check*.yaml linguist-generated
+examples/crds/*.yaml linguist-generated
```

Comment:
is ciliumendpoints.yaml also auto generated files ? If yes, we might need to update `manifest` target in Makefile.

https://github.com/cilium/cilium/blob/master/Makefile#L291

Reply:
Not currently, but it will be in the future.